homeassistant/components/panasonic_viera/config_flow.py

```diff
@@ -31,19 +32,21 @@ class PanasonicVieraConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    _remote: RemoteControl | None = None
```

Comment:
Similar case here. The assignment for instance variables should happen in instance methods (e.g. `__init__`). At best, move the type annotation itself to the class scope.